test_utils/src/authority.rs

```diff
@@ -120,3 +123,27 @@ where
     }
     handles
 }
+
+pub fn create_authority_clients(
```

Comment:
Should this function be called `authority_aggregator` or `test_authority_aggregator`? It seems we have5 or 6 types of clients in the codebase